File: 2015 maj/python/2.py
with open("liczby.txt") as f:
    s = f.read().strip().split('\n')

two, eight = 0, 0
for liczba in s:
    if liczba[-1] == '0':
        two += 1
    
    if liczba[-3:] == '000':
        eight += 1
print(two, eight)

File: 2015 maj/c++/2.cpp
#include <iostream>
#include <fstream>
using namespace std;


int main()
{
    ifstream in("liczby.txt");

    string s;
    int two = 0, eight = 0;
    for (int line = 1; line <= 1000; line++)
    {
        in >> s;
        if (s[s.size() - 1] == '0')
            two++;
        if (s.size() > 3 && s[s.size() - 1] == '0' && s[s.size() - 2] == '0' && s[s.size() - 3] == '0')
            eight++;
    }
    cout << two << ' ' << eight << '\n';
}